test/kennel/models/record_test.rb

```diff
@@ -55,42 +55,6 @@ def self.parse_url(*)
       record.as_json.must_equal some_json
     end
 
-    it "throws if build_json throws, and there were no validation errors" do
```

Comment:
Since we're mostly only _removing_ code in this PR, and the purpose of the PR is to make backtraces more visible, where is the test which proves that we've achieved that goal?

Reply:
I usually don't add tests for behavior that does not exist, i.e. "it does not suppress errors" since that would be then in theory be needed for every method
... I'll verify that backtraces look sane though

Reply:
yeah it raises cleanly as expected